Mathlib/Combinatorics/SimpleGraph/Basic.lean

```diff
@@ -27,46 +23,15 @@ finitely many vertices.
 
 * `SimpleGraph.commonNeighbors` is the intersection of the neighbor sets of two given vertices
 
-* `SimpleGraph.neighborFinset` is the `Finset` of vertices adjacent to a given vertex,
-   if `neighborSet` is finite
-
 * `SimpleGraph.incidenceSet` is the `Set` of edges containing a given vertex
 
-* `SimpleGraph.incidenceFinset` is the `Finset` of edges containing a given vertex,
-   if `incidenceSet` is finite
-
 * `SimpleGraph.Dart` is an ordered pair of adjacent vertices, thought of as being an
   orientated edge. These are also known as "half-edges" or "bonds."
 
-* `SimpleGraph.Hom`, `SimpleGraph.Embedding`, and `SimpleGraph.Iso` for graph
-  homomorphisms, graph embeddings, and
-  graph isomorphisms. Note that a graph embedding is a stronger notion than an
-  injective graph homomorphism, since its image is an induced subgraph.
-
 * `CompleteAtomicBooleanAlgebra` instance: Under the subgraph relation, `SimpleGraph` forms a
   `CompleteAtomicBooleanAlgebra`. In other words, this is the complete lattice of spanning subgraphs
   of the complete graph.
 
-## Notations
-
-* `→g`, `↪g`, and `≃g` for graph homomorphisms, graph embeddings, and graph isomorphisms,
-  respectively.
```

Comment:
Please move this section into Maps

Reply:
That documentation on the notation was already written into the description of `.Maps` when I split that out of `.Basic` in #9579. Also I've just landed in Taiwan for a holiday and may not be able to make all requested changes before returning on Wednesday.

Reply:
Never mind @kmill, I resolved them all!